deploy/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN  apt-get install -y python3-pip git nginx libmariadb-dev-compat
 
 WORKDIR /opt
 
-RUN git clone https://github.com/amlight/Services-Status.git
+RUN git clone https://github.com/Auwate/Services-Status.git
```

Comment:
Hi Austin, this particular change here does not seems necessary. Can you please clarify why we need this?

Reply:
Ah I was testing using my forked repo, I'll fix that right now!